src/style/mixins.scss

```diff
@@ -128,3 +128,12 @@
     }
   }
 }
+
+/// Simulates display: none but keeps the element in DOM tree
+@mixin iui-display-none {
+  opacity: 0;
+  height: 0;
+  width: 0;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
[WebAIM claims](https://webaim.org/techniques/css/invisiblecontent/#:~:text=all%20these%20techniques%20may%20result%20in%20search%20engine%20penalties%20as%20they%20may%20be%20interpreted%20as%20malicious.) that setting width & height to 0
>  may result in search engine penalties as they may be interpreted as malicious.

I know that we use `visibility: hidden;` on modal.scss and are able to still make it animate in/out using a `transition-delay: 0s;`. Does it make sense to do that here as well?

Reply:
`transition-delay` is 0s by default so that won't change anything.

Setting `visibility: hidden` still requires setting dimensions.

Also, we are already setting width & height to 0 in some other places (e.g. checkboxes and toggle switches).

Maybe we can go with 0.1px if we want the animation? Otherwise `display: none` is fine I guess